canary/canary_test.py

```diff
@@ -0,0 +1,82 @@
+import asyncio
+import datetime
+import os
+import time
+
+from pymongo import MongoClient
+from typing import Coroutine, Any
+from viam.robot.client import RobotClient, DialOptions
+from viam.components.board import Board
+
+async def connect(robot_address: str, api_key: str, api_key_id: str) -> Coroutine[Any, Any, RobotClient]:
+    opts = RobotClient.Options(
+        refresh_interval=0,     	 
+        check_connection_interval=0,
+        attempt_reconnect_interval=0,
+        disable_sessions=True,
+        dial_options=DialOptions.with_api_key(api_key_id=api_key_id,api_key=api_key)
+    )
+    return await RobotClient.at_address(robot_address, opts)
+
+async def main():
+    mongo_connection_str = os.environ["MONGODB_TEST_OUTPUT_URI"]
+    db_client = MongoClient(mongo_connection_str)
+    db = db_client["esp32_canary"]
+    coll = db["hourly_results"]
```

Comment:
NIT: something raw_results will be better if we decide to run it more or less frequently